Bigtable/src/Instance.php

```diff
@@ -373,6 +380,27 @@ public function table($id)
         );
     }
 
+    /**
+     * Manage the instance IAM policy
+     *
+     * Example:
+     * ```
+     * $iam = $instance->iam();
```

Comment:
@fiboknacky the tests are failing because this needs a snippet test.

Reply:
Thanks. Added testIam() to Snippet/InstanceTest

Reply:
Good to go now, it seems.